src/main/java/uk/gov/objecthash/TimestampValue.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.objecthash;
+
+import java.time.Instant;
+
+public class TimestampValue implements ObjectHashable {
+
+    private final Instant value;
+
+    public TimestampValue(Instant value) {
```

Comment:
this `public` modifier, I think can be removed?

Reply:
will this be accessible outside of the package? Atm in order to hash a Map you need to build it up from primitives

eg

```java
        Map<String, ObjectHashable> data = new HashMap<>();
        data.put("foo", new StringValue("bar"));
        data.put("null", null);
```

Reply:
Ah, OK I wasn't sure if this was intended to be part of the public API?